### DISCUSSION TITLE
Buzz's Issue in translating and exporting audio, srt file

### DISCUSSION THREAD

**Discussion by lt700** [chidiwilliams/buzz#763] [General]
Hi there,
I have just bought Buzz app from Apple App store and tried with a movie. when I select to a movie to translate that movie into Vietnamese, it showed srt in English, and not exported audio file into Vietnamese, it exported into chinese original language.

Could you pls fix up the issue? the movie must be translated in Vietnamese when I select Vietnamese language, and export to Vietnamese audio, and srt into Vietnamese.

I hope to receive your feedback ASAP and fix up this issue. 
If you don't fix up this and update for me, your app I bought from you seems to be useless for me, waste money.

I have also sent you email, hope you fix it and sent me the update one.

Many thanks,

**Comment by raivisdejus** [comment]
According to a note on [Whisper repo](https://github.com/openai/whisper/discussions/649) translations to non-English were never initial part of their training objective, so this feature may be unreliable. 

An approach you could try is to use "Huggingface" whisper engine and specify `openai/whisper-large` as the model. That should use the older model version that may lead to better results.
